.github/workflows/workflow.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         os: [ubuntu-latest]
         node-version: ['14.18.0', '*']
         # Must include the minimum deno version from the `DENO_VERSION_RANGE` constant in `node/bridge.ts`.
-        deno-version: ['v1.32.5', 'v1.x']
+        deno-version: ['v1.37.0', 'v1.x']
```

Comment:
Updating to the latest bootstrap means that we now rely on `Deno.serve` to run tests, which wasn't available on 1.32.5.